src/_guides/language/language-tour.md

```diff
@@ -2470,9 +2470,9 @@ When exactly do assertions work?
 That depends on the tools and framework you're using:
 
 * Flutter enables assertions in [debug mode.][Flutter debug mode]
-* Development-only tools such as [dartdevc][]
+* Development-only tools such as [`webdev serve`][]
   typically enable assertions by default.
-* Some tools, such as [`dart run`][] and [dart2js][]
+* Some tools, such as [`dart run`][] and [`dart compile`][]
   support assertions through a command-line flag: `--enable-asserts`.
```

Comment:
I don't think `dart compile` has the `--enable-asserts` flag?

Reply:
Maybe? See [dart2js code](https://github.com/dart-lang/sdk/blob/main/sdk/bin/dart2js#L40-L44).

Reply:
We have it for the dart2js subcommand (`dart compile js`), but I don't know about all the other subcommands under `dart compile`. For example `dart compile exe` seems to have it, but jit-snapshot and kernel don't seem to have it.